impf/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class WorkflowException(Exception):
+    pass
```

Comment:
We should move the API Exceptions in `constructor` to `exceptions.py` too

Reply:
Done